### PR TITLE
[Chore] Update readme for tooling and tutorial 1

### PR DESCRIPTION
**Pull request by suho**
## What happened 👀

Update README.md to link to the wiki page and add tutorial1/README.md
 
## Insight 📝

- Update README.md to link to the tooling wiki page
- Add README.md to link to the tutorial 1 wiki page
 
## Proof Of Work 📹

N/A

### CODE REVIEW

File: tooling/README.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ## VIPER Xcode Templates
 
-We have created Xcode templates to generate all needed VIPER scaffolded classes, making VIPER usage and adoption easier and ready to add business logic.
+Nimble iOS developers have created Xcode templates to generate all needed VIPER scaffolding classes.
 
-For details on installation and usage, please see wiki page [WIP].
\ No newline at end of file
+For details, please see on [tooling wiki page](https://github.com/nimblehq/viper-demo/wiki/Tooling)
```
Comment (blyscuit) [inline tooling/README.md:5]:
```suggestion
For details, please see [Tooling wiki page](https://github.com/nimblehq/viper-demo/wiki/Tooling)
```

File: tutorials/tutorial1/README.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# Tutorial 1: Creating a VIPER module
+
+This tutorial will help you understand a VIPER module.
+
+## Getting started
+
+Install tooling, unless you've done already:
```
Comment (blyscuit) [inline tutorials/tutorial1/README.md:7]:
```suggestion
Install the tools, unless you've done already:
```

Comment (blyscuit) [reply tutorials/tutorial1/README.md:7]: tooling works too.

**Review by blyscuit — APPROVED** [review]

**Review by markgravity — APPROVED** [review]
LGTM 🤖

**Pull request merged**